platzky/admin/fake_login.py

```
@@ -8,23 +8,30 @@
 
-from flask import Blueprint, flash, redirect, session, url_for
+from flask import Blueprint, flash, redirect, session, url_for, render_template_string
+from flask_wtf import FlaskForm
 from markupsafe import Markup
 
 ROLE_ADMIN = "admin"
 ROLE_NONADMIN = "nonadmin"
 VALID_ROLES = [ROLE_ADMIN, ROLE_NONADMIN]
 
 
+class FakeLoginForm(FlaskForm):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)